lib/ip.js

```diff
@@ -376,7 +376,25 @@ ip.address = function(name, family) {
     return res[0].address;
   }
 
-  var all = Object.keys(interfaces).map(function (nic) {
+  function priority (name) {
```

Comment:
should remove space before `(name` for consistency with the rest of the codebase